distribution/src/config/jvm.options

```diff
@@ -33,16 +33,18 @@
 ################################################################
 
 ## GC configuration
--XX:+UseConcMarkSweepGC
--XX:CMSInitiatingOccupancyFraction=75
--XX:+UseCMSInitiatingOccupancyOnly
+8-13:-XX:+UseConcMarkSweepGC
+8-13:-XX:CMSInitiatingOccupancyFraction=75
+8-13:-XX:+UseCMSInitiatingOccupancyOnly
 
-## G1GC Configuration; to use G1GC uncomment the lines below
-# -XX:-UseConcMarkSweepGC
-# -XX:-UseCMSInitiatingOccupancyOnly
-# -XX:+UseG1GC
-# -XX:G1ReservePercent=25
-# -XX:InitiatingHeapOccupancyPercent=30
+## G1GC Configuration
+# to use G1GC, uncomment the next line and update the version on the following
+# lines to your version of the JDK
+# 8-13:-XX:-UseConcMarkSweepGC
+14-:-XX:-UseCMSInitiatingOccupancyOnly
```

Comment:
Isn't this option only relevant when CMS exists?

Reply:
Oops, I missed that. Thanks.

Reply:
@rjernst I pushed d282f032e3655506eca0372674f4c10283c914c6.